app/serverconfig.ini

```diff
@@ -0,0 +1,5 @@
+[database]
+location = localhost
```

Comment:
Can you call this argument "host" instead?

Reply:
sure